index.html

```diff
@@ -2,90 +2,93 @@
-    <div class="links">
-      <div class="link">
-        <a href="https://www.linkedin.com/in/jlcarmic/" target="_blank">LinkedIn</a>
-      </div>
-      <div class="link">
-        <a href="https://github.com/jlcarmic" target="_blank">GitHub</a>
-      </div>
-      <div class="link">
-        <a href="https://codepen.io/jlcarmic/" target="_blank">CodePen</a>
-      </div>
-      <div class="link">
-        <a href="./resume.pdf" target="_blank">Resume</a>
-      </div>
-    </div>
-  </header>
-  <div class="content">
-    <div class="section about">
-      <div class="photo"></div>
-      <div class="description">
-        <p>
-          I am a product developer with over a decade of experience building best-of-class solutions.
-          With expertise in product development, software engineering, cyber security, and technical operations,
-          I have helped startups create products to delight their customers.
-        </p>
-        <p>
-          I have held positions as a Software Engineer, Security Engineer, Director of Operations, and Vice President
-          of Product & Engineering.
-        </p>
-      </div>
-    </div>
-    <div class="divider">Projects</div>
-    <div class="section projects">
-      <div class="project">
-        <div class="explanation">
-          <p>Animated floating Hal Jordan, the Green Lantern.</p>
-          <p>Image is an SVG created in Sketch.</p>
-          <p>Animation is achieved with CSS.</p>
-          <a href="https://github.com/jlcarmic/green-lantern" target="_blank">
-            <button class="button">Green Lantern</button>
-          </a>
-        </div>
-        <div class="screenshot">
-          <img src="images/greenlantern.png" />
-        </div>
-      </div>
+  <div>
+    <header class="clearfix">
+      <h1>Patrick Langdon O'Connor</h1>
 
-      <div class="project">
-        <div class="screenshot">
-          <img src="./images/producthuntalexa.png" />
-        </div>
-        <div class="explanation">
-          <p>An Alexa Skill for hearing Today's Featured products.</p>
-          <p>Built using NodeJS and AWS Lambda.</p>
-          <p>Utilizes ProductHunt's API.</p>
-          </p>
-          <a href="https://github.com/jlcarmic/producthunt_alexa_skill" target="_blank">
-            <button class="button">Today's Featured</button>
-          </a>
-        </div>
-      </div>
+      <img src="images/me.jpg" class="me-img" width="300" height="300">
+      <p class="tagline">Junior back-end developer driven by curiosity</p>
+      <address>
+        <p><strong>Email:&nbsp</strong><a href="mailto:poc.967@gmail.com">poc.967@gmail.com</a></p>
+        <p><strong>Cell:&nbsp</strong><a href="tel:5082590690">508-259-0690</a></p>
+        <p><strong>Home:&nbsp</strong><a href="tel:5084291838">508-429-1838</a></p>
+      </address>
+      <ul class="nav">
+        <li><a href="https://www.linkedin.com/in/patrickoconnor3/">LinkedIn</a></li>
```

Comment:
The convention is when linking to external sites to always throw a new tab with `target="_blank"`